bootstrap.sh

```diff
@@ -1,20 +1,22 @@
 #!/bin/bash
-kubectl apply -f .infrastructure/mysql/ns.yml
-kubectl apply -f .infrastructure/mysql/configMap.yml
-kubectl apply -f .infrastructure/mysql/secret.yml
-kubectl apply -f .infrastructure/mysql/service.yml
-kubectl apply -f .infrastructure/mysql/statefulSet.yml
+kubectl apply -f ./.infrastructure/mysql/ns.yml
+kubectl apply -f ./.infrastructure/mysql/configMap.yml
+kubectl apply -f ./.infrastructure/mysql/secret.yml
+kubectl apply -f ./.infrastructure/mysql/service.yml
+kubectl apply -f ./.infrastructure/mysql/statefulSet.yml
 
-kubectl apply -f .infrastructure/app/ns.yml
-kubectl apply -f .infrastructure/app/pv.yml
-kubectl apply -f .infrastructure/app/pvc.yml
-kubectl apply -f .infrastructure/app/secret.yml
-kubectl apply -f .infrastructure/app/configMap.yml
-kubectl apply -f .infrastructure/app/clusterIp.yml
-kubectl apply -f .infrastructure/app/nodeport.yml
-kubectl apply -f .infrastructure/app/hpa.yml
-kubectl apply -f .infrastructure/app/deployment.yml
+kubectl apply -f ./.infrastructure/app/ns.yml
+kubectl apply -f ./.infrastructure/app/pv.yml
+kubectl apply -f ./.infrastructure/app/pvc.yml
+kubectl apply -f ./.infrastructure/app/secret.yml
+kubectl apply -f ./.infrastructure/app/configMap.yml
+kubectl apply -f ./.infrastructure/app/clusterIp.yml
+kubectl apply -f ./.infrastructure/app/nodeport.yml
+kubectl apply -f ./.infrastructure/app/hpa.yml
+kubectl apply -f ./.infrastructure/app/deployment.yml
 
 # Install Ingress Controller
 kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/main/deploy/static/provider/kind/deploy.yaml
 # kubectl apply -f .infrastructure/ingress/ingress.yml
+
```

Comment:
The commented-out line `# kubectl apply -f .infrastructure/ingress/ingress.yml` seems redundant since the next line executes the same command. Consider removing this line if it's not needed for documentation or future reference.